src/main/webapp/i18n/de/lecture.json

```diff
@@ -31,6 +31,7 @@
             "endDate": "Ende",
             "visibleDate": "Sichtbar ab",
             "course": "Kurs",
+            "ingestionState": "Vorlesungsimport in Pyris",
```

Comment:
Can you change this translation to be more similar to the English version? It should be clear at least that the following information is a state